src/main/java/seedu/duke/Duke.java

```diff
@@ -1,21 +1,41 @@
 package seedu.duke;
 
-import java.util.Scanner;
+import diet.dietmanager.DietManager;
+import diet.dietmanager.DietManagerUI;
 
 public class Duke {
     /**
      * Main entry-point for the java.duke.Duke application.
      */
+
+    private final DietManagerUI dietManagerUI;
+    private final DietManager dietManager;
+    public Duke() {
+        dietManagerUI = new DietManagerUI();
+        dietManager = new DietManager();
+
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-        System.out.println("What is your name?");
-
-        Scanner in = new Scanner(System.in);
-        System.out.println("Hello " + in.nextLine());
+        new Duke().run();
+    }
+
+    public void run() {
+        System.out.println("Hi, this is the Schwarzenegger, " +
+                "an app for your daily workout and diet recording.");
+
+        execute();
+    }
+
+    public void execute() {
+        String response = dietManagerUI.getInput();
+        while (!response.equals("exit")) {
+
```

Comment:
Try use commandLibrary as well?

Reply:
not sure whether commLib will be more convenient or not if we only have "diet", "workout", "profile" and "exit" commands which don't really execute many things. Maybe a simple switch case would be clearer?